src/app/main/dashboard/dashboard-card/dashboard-card.component.scss

```diff
@@ -1,10 +0,0 @@
-:host {
```

Comment:
Removing these styles causes to cards to size themself in respect to their content, whcih would make them differently sized, when they have differently sized content.
Would be better when the styles remain.